backend/api/Areas/Property/Models/Search/PropertyModel.cs

```diff
@@ -52,12 +43,12 @@ public class PropertyModel
         /// <summary>
         /// get/set - The GIS latitude location of the property.
         /// </summary>
-        public double Latitude { get; set; }
+        public double? Latitude { get; set; }
```

Comment:
Nulled values will not get returned in the JSON from the API by default.  Which will reduce the clutter of the flattening process of returning parcel and building properties in a single model.